rust/actyx/ax/src/cmd/mod.rs

```diff
@@ -48,15 +47,20 @@ pub struct ConsoleOpt {
     authority: Authority,
     /// File from which the identity (private key) for authentication is read.
```

Comment:
ISTR this also stating that the key can be given verbatim — in any case, could you add that? And while you’re at it, could you copy the doc string to all uses below?